.github/workflows/build_and_deploy.yml

```
@@ -34,41 +34,49 @@ jobs:
             features: ""
             target: x86_64-pc-windows-msvc
             artifact_name: windows-x64-cpu
+            whl_package_name: voicevox-core-cpu
             use_cuda: false
           - os: windows-latest
             features: directml
             target: x86_64-pc-windows-msvc
             artifact_name: windows-x64-directml
+            whl_package_name: voicevox-core-directml
             use_cuda: false
           - os: windows-latest
             features: ""
             target: x86_64-pc-windows-msvc
             artifact_name: windows-x64-cuda
+            whl_package_name: voicevox-core-cuda
             use_cuda: true
           - os: windows-latest
             features: ""
             target: i686-pc-windows-msvc
             artifact_name: windows-x86-cpu
+            whl_package_name: voicevox-core-cpu
             use_cuda: false
           - os: ubuntu-latest
             features: ""
             target: x86_64-unknown-linux-gnu
             artifact_name: linux-x64-cpu
+            whl_package_name: voicevox-core-cpu
             use_cuda: false
           - os: ubuntu-latest
             features: ""
             target: x86_64-unknown-linux-gnu
             artifact_name: linux-x64-gpu
+            whl_package_name: voicevox-core-gpu
```

windowsのCUDA版と揃えてこうするのはどうでしょう
```suggestion
            whl_package_name: voicevox-core-cuda
```